### PR TITLE
Portfolio:Mobile version

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this milestone, I achieved the following:**

- Built a Mobile version of my portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                </ul>
+            </div>
+
+
+
+        </section>
+    
+        
+    
+
```
Comment (TedLivist) [inline index.html:52]: - [ ] Please remove unnecessary blank lines as provided in the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). This makes for code readability and organization. Also, do this for other parts of your code.

blank_m

File: index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                        <li class="bootstrap">bootstrap</li>
+                        <li>Ruby</li>
+                       </ul>
+                </div>
+
+                <a href="#">See Project</a>
+            </div>
+
+                <div class="projects">
+                    <div class="project">
```
Comment (TedLivist) [inline index.html:79]: - [ ] Please also fix your indentations as provided in the best practices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+            <div class="feature-project">
+                
+                <div class="phimg">
+                    <img src="./images/placeholder.png" alt="placeholder-img">
+                </div>
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                         industry's standard dummy text ever since the 1500s, when an unknown printer took a 
+                         standard dummy text.
+                    </p>
```
Comment (TedLivist) [inline index.html:64]: - [ ] Please fix your indentations across the code too.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html</li>
+                                <li class="project-language"> Bootstrap</li>
+                                <li class="project-language"> Ruby</li>
+                            </ul>
+                        </div>
+                        
+                            <a href="#" class="project-btn">See project</a>
```
Comment (TedLivist) [inline index.html:93]: - [ ] As defined in the Figma template your `See Project` buttons/links should have stylings for `:active` and `:hover`

btns-styles

File: README.md [hunk 5]
```diff
@@ -1 +1,39 @@
-# Portfolio
\ No newline at end of file
+# My-Portfolio
+This is my second microverse project. In this project I built my portfolio(Mobile version).Thismarks the second step for the portfolio project.The main objective for this project was to build template from figma using CSS and Flexbox. I created my-portfolio reposotory and cloned into my local machine. I have implemented linter,Github Flow and professional documentation which is one of the project building requirements at microverse.
+
```
Comment (TedLivist) [inline README.md:3]: - [ ] As presented in the template please make the font-weight of the different section titles - 'My Recent Works', 'Multi-Post Stories' and 'Profesional Art Printing Data' - should be **bolder** than they are.

header-styles

File: index.html [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+
+
+    </main>
+
+
+
+    
+</body>
+
```
Comment (TedLivist) [inline index.html:232]: - [ ] Please fix your indentations and remove unnecessary blank lines.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+                 </div>
+                </section>
+
+            <section class="about">
+                <div class="description">
+                    <h2>About me</h2>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <a href="#">Get My Resume</a>
```
Comment (TedLivist) [inline index.html:187]: - [ ] As mentioned earlier, please add `:hover` and `:active` stylings to your links.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+                   <h3><img src="./images/squareicon.png" class="square-icon" alt="square-icon">Frameworks</h3>
+                   <ul>
+                    <li>bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                   </ul>
+
+               <h3><img src="./images/circleicon.png" class="square-icon" alt="skills-icon">Skills</h3>
```
Comment (TedLivist) [inline index.html:208]: - [ ] Your 'Languages', 'Frameworks' and 'Skills' categories should be inside a `ul`, with each of them in an `li`

langs

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️ 

Hi @Tumworobere,

Good job so far!
- Your UI design is quite top-notch ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (Tumworobere) [reply index.html:208]: Thanks for taking your time to review my project.Am going to work on the requested changes

**Review by mke2111 — APPROVED** [review]
### Hi @Tumworobere, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
I have worked on the following changes as requested by the first reviewer:


> 
> ### Hi @Tumworobere,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> **Congratulations! 🎉**
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks @mke2111  and @TedLivist  for reviewing my project

File: index.html (hunk 7 above)
Comment (Tumworobere) [reply index.html:157]: Alright, @TedLivist This has been noted.Thanks!

**Pull request merged**